File: system-test/sec-tests/python/sec_55_plaintext_password.py
# SEC-55: Sensitive Data Plaintext Storage
# Test: TC-CR-058
import sqlite3

def register_user(username, password):
    conn = sqlite3.connect('app.db')
    conn.execute("INSERT INTO users (username, password) VALUES (?, ?)",
                 (username, password))
    conn.commit()

File: system-test/sec-tests/java/Sec55PlaintextPassword.java
// SEC-55: Sensitive Data Plaintext Storage
// Test: TC-CR-058
package com.example;

import java.sql.*;

public class UserRegistrationService {

    public void register(String username, String password) throws Exception {
        Connection conn = getConnection();
        Statement stmt = conn.createStatement();
        stmt.execute("INSERT INTO users (username, password) VALUES ('"
            + username + "', '" + password + "')");  // Password stored in plaintext
    }
}
